scrape/scrape_test.go

```diff
@@ -374,8 +375,9 @@ func TestScrapePoolRaces(t *testing.T) {
 
 func TestScrapeLoopStopBeforeRun(t *testing.T) {
 	scraper := &testScraper{}
+	ctx := context.Background()
 
-	sl := newScrapeLoop(context.Background(),
+	sl := newScrapeLoop(ctx,
```

Comment:
We don't need a new variable here. Same for all the other places, we can use `context.Background()` directly.